packages/salesforcedx-vscode-lwc/src/testSupport/workspace/index.ts

```diff
@@ -9,9 +9,10 @@ import { getLwcTestRunnerExecutable } from './getLwcTestRunnerExecutable';
 import { getTestWorkspaceFolder } from './getTestWorkspaceFolder';
 import { workspaceService } from './workspaceService';
 
-export {
+const workspace = {
```

Comment:
It's not clear why the export was broken up into a const and an export.  Why was it spilt?

Reply:
It is required.  It comes down to a copy by value vs copy by reference problem. A quick google didn't give me a great link to share, but the gist of it is when we export a stand alone function there is a requirement that the function be stubbed prior to the function ever being loaded by module under test or the stub will not be present when the code under test is executed.  Basically it comes down to something like 
providesFunction.ts
``` 
export function immaFunction() {}
```

usesFunction.ts
```
import {immaFunction} from './functions';

class UsesFunction {
  constructor() { 
      immaFunction();
  }
}
```

usesFunction.test.ts
```
import {UsesFunction} from '../../src/usesFunction';
import * as func from '../../src/functions';
...
beforeEach(() => {
  immaFunctionStub = sandbox.stub(funcs, 'immaFunction');
});

it('Should call the stub.', () => {
  const usesFuncInstance = new UsesFunction();
  // Fails due to immaFunction not being the stubbed during execution
  expect(immaFunctionStub.callCount).to.equal(1);
});
```
Forgive the pseudo code.

b/c UsesFunction is loaded before immaFunction stub is created it loads the original function that was exported and it can not be stubbed without getting crazy.  However if you wrap the functions in an object you are able to stub the property on the object b/c it is a copy of the object and not the value of the function (confusing I know).  This is/was the root of the issue when trying to stub exported singular functions in mocha, and it's still an issue in jest.  

Hit me up in slack if this is unclear and I can take you though what I was seeing locally.